src/apurvaJun20/FindCharacter.java

```diff
@@ -0,0 +1,42 @@
+package apurvaJun20;
+
+public class FindCharacter {
+    void findonechar(String input,char character) {
+		int count=0;
+		for(int index=0;index<input.length();index++) {
+			if(input.charAt(index)==character ) {
+				count++;
+			}
+		}
+		 System.out.println( "Total count of " +character+ " in " +input + " is " +count+"\n" );
+	}
+	void allcharfind(String name){
+		for(int index=0;index<name.length();index++) {
+			findonechar(name,name.charAt(index));
+		}	
+	}
+	void findeachString(String[] name)
+	{ 
+		for(int index=0;index<name.length;index++)
+		{
+		allcharfind(name[index]);
+		}
```

Comment:
i want every operation in one method only. Please make it happen.

Reply:
Could you explain this to me as how it should be